e2e/app/component/workspace-card.ts

```diff
@@ -43,13 +43,12 @@ export default class WorkspaceCard extends CardBase {
    * @throws TimeoutError if fails to find Card.
    */
   static async findAllCards(page: Page, accessLevel?: WorkspaceAccessLevel): Promise<WorkspaceCard[]> {
-    try {
-      await waitWhileLoading(page);
-      await page.waitForXPath(WorkspaceCardSelector.cardRootXpath, { visible: true, timeout: 1000 });
-    } catch (e) {
-      return [];
-    }
```

Comment:
Code is problematic. Say if `waitWhileLoading` has failed, an empty array is returned by function.